### ISSUE TITLE
Compile issue due to picotls change

### ISSUE THREAD

**Issue by rafee** [private-octopus/picoquic#440]
Getting the following error when trying to compile picoquic on Ubuntu 18.10. But as it's related to picotls, I assume it should be same for everyone.
`picoquic/picoquic/tls_api.c:1038:14: error: ‘ptls_context_t’ {aka ‘struct st_ptls_context_t’} has no member named ‘hkdf_label_prefix’; did you mean ‘hkdf_label_prefix__obsolete’`
I tried by changing the name and code compiled successfully

**Comment by huitema** [comment]
One simple way to solve your problem would be to use the version of picotls that we verified to work in our tests. Look at the file ci/build_picotls.sh in the repo. Instead of just cloning the latest version of Picotls, it does:
```
# Build at a known-good commit
COMMIT_ID=241f684346d3be4f5ba8dc46010e9f9486a79991

cd ..
git clone --branch master --single-branch --shallow-submodules --recurse-submodules --no-tags https://github.com/h2o/picotls
cd picotls
git checkout "$COMMIT_ID"
#git submodule init
#git submodule update
cmake .
make -j$(nproc) all
cd ..
```
We did that because picotls is a living project, it changes, and some changes are bound to break us.

That said, I clearly need to work with Kazuho to understand the latest changes in Picotls and move the commit point forward. The `hkdf_label_prefix` issue is probably linked to the evolution of the interface between QUIC and TLS, as specified in QUIC draft-17. We are going to implement draft-17 soon, and resolve that.

**Comment by huitema** [comment]
Fixed by PR #443
